### ISSUE TITLE
Комментарии к заявке

### ISSUE THREAD

**Issue by aakuptsov** [BellyJam/28DL#1]
1. В имя файла с заявкой добавить фамилию и знак подчёркивания.

2. В игре обычно есть некто, кто противостоит главному герою, какие-то враги, препятствия и т. п., мешающие ему выполнить его задачу. Кто будет врагом здесь? Понятие врага в данном случае весьма размыто - это кто угодно или что угодно, что мешает герою достичь его цели.

3. Какие именно элементы вселенной Лицея будут присутствовать?

**Comment by coralf0x** [comment]
**_> 2. В игре обычно есть некто, кто противостоит главному герою, какие-то враги, препятствия, мешающие ему выполнить его задачу. Кто будет врагом здесь? Понятие врага в данном случае весьма размыто - это кто угодно или что угодно, что мешает герою достичь его цели._**
Препятствий и трудностей в Лицее предостаточно. Обязательно ли в заявке расписывать всех НПС, врагов? Я не могу работать над этим по ходу разработки?

**_> 3. Какие именно элементы вселенной Лицея будут присутствовать?_**
Все локации будут либо внутри лицея, либо связаны с ним. Персонажи, предметы, задания. Также не понимаю, зачем прописывать эту информацию в заявке. Мне кажется ни один инди-разработчик до начала разработки не знает конечного результата и что он оставит, а что уберет.

**Comment by aakuptsov** [comment]
>>Обязательно ли в заявке расписывать всех НПС, врагов? Я не могу работать над этим по ходу разработки?

Нет, расписывать необязательно сейчас, вопрос просто на понимание того, что будет происходить. Расписать надо будет в сценариях.

Согласен с тем, что сразу от начала до конца трудно знать, что будет в игре.
